todo_app/app.py

```diff
@@ -37,6 +43,12 @@ def writer_check(*args, **kwargs):
 def create_app():
     app = Flask(__name__)
     app.config.from_object(Config())
+    logger = logging.getLogger(__name__)
+
```

Comment:
I think you've skipped Step 3: setting the log level - happy for you to apply this before merging since the code is given but we should use this since we've configured the environment variable in Terraform! I've assumed in the snippet below that you'll read the environment variable directly, and set a default of INFO to match your Terraform default. Try switching this variable and check that it only logs the relevant messages
```
app.logger.setLevel(os.getenv('LOG_LEVEL', 'INFO'))
```

Reply:
Added to line 47